### ISSUE TITLE
Create distribution package in FreeBSD

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster#835]
To be able to create a the Zonemaster-LDNS package (`perl Makefile.PL --no-ed25519`, `make all`, `make dist`) in a FreeBSD server the following packages must be installed (e.g. using the following command):

```
pkg install git-lite perl5 p5-Module-Install p5-Devel-CheckLib p5-Module-Install-XSUtil libidn devel/p5-Test-Fatal devel/Libtool devel/autoconf devel/automake
```

Create or update some document to include this information, which is by no means obvious.

**Comment by mattias-p** [comment]
I don't think we have a document where this information would fit. I propose we create a new document at `docs/internal-documentation/maintenance/BuildSystemPreparation.md`.

**Comment by matsduf** [comment]
Right now I try to document to see how it works.
